src/TinyMCEInput.js

```diff
@@ -82,7 +82,8 @@ var TinyMCEInput = React.createClass({
       pollInterval: 1000,
       textareaProps: {},
       otherEventHandlers: {},
-      onChange: function() {}
+      onChange: function() {},
+      component: 'textinput',
```

Comment:
Just verified in my project that 1) This is a breaking change (as in it breaks default functionality of this package), and 2) Setting it back to "textarea" fixes it.